packages/meditrak-server/src/tests/routes/GETMapOverlayGroupRelations.test.js

```diff
@@ -0,0 +1,151 @@
+/**
```

Comment:
I'm aware that `GETMapOverlay.test.js`, `GETMapOverlayGroup.test.js` and `GETMapOverlayGroupRelation.test.js` have a lot of duplicated code. But I'm thinking that this is because they just really have similar permission model. Would totally understand if you think it needs to be made less DRY and happy to do some more work to refactor them if that's the case

Reply:
No need to do so in this PR - we could decide to do this on a larger scale at some point, if it makes sense